README.md

```diff
@@ -5,9 +5,8 @@ The `solarwinds-apm` module provides [SolarWinds](https://www.solarwinds.com) in
 It supports most commonly used databases, frameworks, and packages automatically. An
 API allows anything to be instrumented.
 
-An [SolarWinds](https://www.solarwinds.com) account is required to view metrics.
-Accounts are [free](https://www.solarwinds.com) for development
-and testing use. For production usage a [free trial](https://www.solarwinds.com)
+A [SolarWinds](https://www.solarwinds.com) account is required to view traces & metrics.
+Accounts are [free](https://www.solarwinds.com) for development and testing use. For production usage a [free trial](https://www.solarwinds.com)
```

Comment:
@ronilan can you remove this line about the dev/testing and production account? It will likely be different from AppOptics.

Reply:
Done